src/oas2/guards.ts

```diff
@@ -15,6 +16,18 @@ export const isTagObject = (maybeTagObject: unknown): maybeTagObject is Tag => {
   return false;
 };
 
+export const isXLogo = (maybeXLogo: unknown): maybeXLogo is NonNullable<IHttpService['logo']> => {
+  if (isDictionary(maybeXLogo)) {
+    maybeXLogo.altText = isString(maybeXLogo.altText) ? maybeXLogo.altText : 'logo';
```

Comment:
Hmm, I would say that a type guard shouldn't alter the data.
It's rather a side effect you wouldn't expect from that kind of a function.